components/ws-proxy/pkg/proxy/config.go

```diff
@@ -71,10 +71,11 @@ func (c *HostBasedIngressConfig) Validate() error {
 
 // WorkspacePodConfig contains config around the workspace pod.
 type WorkspacePodConfig struct {
-	TheiaPort           uint16 `json:"theiaPort"`
-	IDEDebugPort        uint16 `json:"ideDebugPort"`
-	SupervisorPort      uint16 `json:"supervisorPort"`
-	SupervisorDebugPort uint16 `json:"supervisorDebugPort"`
+	TheiaPort               uint16 `json:"theiaPort"`
+	IDEDebugPort            uint16 `json:"ideDebugPort"`
+	SupervisorPort          uint16 `json:"supervisorPort"`
+	SupervisorDebugPort     uint16 `json:"supervisorDebugPort"`
+	DebugWorkspaceProxyPort uint16 `json:"debugWorkspaceProxyPort"`
```

Comment:
Is it always one? I wonder whether or not a user could create multiple debug workspaces.

Reply:
Yes, it can be only one.